src/shortcuts/__tests__/undo.ts

```diff
@@ -208,13 +206,6 @@ it('undo contiguous changes', () => {
       context: [ROOT_TOKEN],
       thoughtsRanked: [{ value: 'A', rank: 0 }]
     },
-    {
-      type: 'existingThoughtChange',
-      newValue: 'Atlantic ',
-      oldValue: 'Atlantic',
-      context: [ROOT_TOKEN],
-      thoughtsRanked: [{ value: 'Atlantic', rank: 0 }]
-    },
```

Comment:
You need to change `oldValue` in the next `existingThoughtChange` to `Atlantic` instead of `Atlantic `.